doc/Doxyfile

```diff
@@ -781,8 +819,7 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = src \
-                         lib
+INPUT                  = src
```

Comment:
this looks right, just making sure. I would have separated the 'autogenerated changes' doxygen file and the 'manual settings change' into different commits, for easy review.